Add documentation on storage partitioning and partitioned cookies (#6968)
GoogleChrome/developer.chrome.com: site/en/docs/extensions/reference/cookies/index.md

}
```

## Partitioning {: #partitioning }

[Partitioned cookies][chips] allow a site to mark that certain cookies should be keyed against the
origin of the top level frame. This means that if site A is embedded using an iframe in site B
and site C, a partitioned cookie can have a different value in each.

> `chrome.cookies` does not currently support partitioning, which means that all methods

Reviewer: ```suggestion
`chrome.cookies` does not support partitioning, which means that all methods
```

read and write cookies from all partitions. The [`cookies.set()`](#method-set) method stores cookies in
the default partition.

> For more details on the general impact of partitioning for extensions, see the
> [Storage and Cookies][cookie-partitioning] page.

Reviewer: ```suggestion
For details on the general impact of partitioning for extensions, see
[Storage and Cookies][cookie-partitioning].
```

}
```

## Partitioning {: #partitioning }

{% Aside %}

> We are currently working on changes to the API to support partitioned cookies. For more information,

Reviewer: Please don't put future plans in documentation for established features. The corporate style guide strongly discourages this, but doesn't say why. I can think of at least two problems.
* We're very unlikely to remember to remove it at the appropriate time.
* The developed feature may or may not resemble what we stated.

> We are currently working on changes to the API to support partitioned cookies. For more information,

Author: Updated this per our discussion :)

> {% Aside %}
> We recently published an [API proposal][partitioning-api-proposal] for changes to the
> `chrome.cookies` API that would improve support for partitioning.
> {% endAside %}
> 

Reviewer: ```suggestion
```

Reviewer: https://developers.google.com/style/timeless-documentation

Author: We specifically discussed this wording in our meeting (changed from previous wording which said we were actively working on changes). With that context I'd like to keep this.

Author: Removed this.

[Partitioned cookies][chips] allow a site to mark that certain cookies should be keyed against the
origin of the top level frame. This means that if site A is embedded using an iframe in site B
and site C, a partitioned cookie can have a different value in each.

> `chrome.cookies` does not currently have support for partitioning, which means that all methods

Reviewer: ```suggestion
`chrome.cookies` does not currently support partitioning, which means that all methods
```

> consider cookies from all partitions. The [`cookies.set()`](#method-set) method stores cookies in

Reviewer: ```suggestion
read and write cookies from all partitions. The [`cookies.set()`](#method-set) method stores cookies in
```
